etc/config-template.toml

```diff
@@ -137,6 +137,15 @@
 [pd]
 ## PD endpoints.
 # endpoints = []
+# The interval at which to retry a PD connection initialization. Set to 0 to disable retrying 
+# connection initialization.
+# retry-interval = 300
```

Comment:
I prefer "300ms" here

Reply:
OK! 👍 We can use http://tailhook.github.io/humantime/humantime/fn.parse_duration.html